route4me-csharp-sdk/Route4MeSDKLibrary/Route4MeManager.cs

```diff
@@ -6833,10 +6833,37 @@ public TerritoryZone UpdateTerritory(TerritoryZoneParameters territoryParameters
 
                                 try
                                 {
-                                    var streamTask = ((StreamContent) response.Result.Content).ReadAsStreamAsync();
-                                    streamTask.Wait();
-
-                                    errorResponse = streamTask.Result.ReadObject<ErrorResponse>();
+                                    errorResponse = default; 
```

Comment:
there is change for GetJsonObjectFromAPI, but there is no change for GetJsonObjectFromAPIAsync

Also GetXmlObjectFromAPI/GetXmlObjectFromAPIAsync is not considered